config/src/config/state_sync_config.rs

```diff
@@ -296,6 +296,33 @@ impl Default for AptosDataPollerConfig {
     }
 }
 
+#[derive(Clone, Copy, Debug, Deserialize, PartialEq, Eq, Serialize)]
+#[serde(default, deny_unknown_fields)]
+pub struct AptosDataMultiFetchConfig {
+    /// Whether or not to enable multi-fetch for data client requests
+    pub enable_multi_fetch: bool,
+    /// The number of additional requests to send per peer bucket
```

Comment:
SGTM, I added this to the config:
```
    /// The number of peers per multi-fetch bucket. We use buckets
    /// to track the number of peers that can service a multi-fetch
    /// request and determine the number of requests to send based on
    /// the configured min, max and additional requests per bucket.
    pub multi_fetch_peer_bucket_size: usize,
```

Does that make more sense?

Reply:
Shall we allow the client sending out the additional requests only when the first one didn't come back within some threshold?

Reply:
My feeling is that it's much more performant to actually just send them out together (all at once) rather than to wait for a (mini) timeout before sending the second request 😄 Plus, most of the node's network connections are heavily under-utilized, so I'm not very worried about the increased network resources because we'll be load balancing them across TCP connections.

Note: one could emulate the mini-timeout idea today already by just setting much lower RPC timeouts and relying on exponential backoff to kick-in. But, I think it's probably unnecessary.